### PR TITLE
fix: parse the isProduction flag from iOS variant secret

### PR DESCRIPTION
**Pull request by darahayes**
Related issue: https://issues.jboss.org/browse/AEROGEAR-2562

## Verification Steps

Use [this branch of the unifiedpush-apb](https://github.com/aerogearcatalog/unifiedpush-apb/pull/49) to provision the service. Note that the isProduction checkbox should result in the appropriate variant being created UPS.

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -371,13 +382,15 @@ func pushClientOrDie() *upsClient {
...
 		serviceInstanceId: serviceInstanceId,
-		baseUrl: upsBaseURL,
+		baseUrl:           upsBaseURL,
 	}
 }
 
 func main() {
 	rand.Seed(time.Now().Unix())
 
+	log.Print("Hello this is Dara's custom service")
```
Comment (pb82) [inline cmd/server/main.go]: I think you can remove that log ;)

Comment (darahayes) [reply cmd/server/main.go]: Good catch! A nice little sanity check to make sure I was actually running my code 😄

**Comment by darahayes** [comment]
@pb82 I've removed that log statement :)

**Comment by pb82** [comment]
👀

**Pull request merged**